### DISCUSSION TITLE
Rename devices

### DISCUSSION THREAD

**Discussion by ymurawski** [OpenCCU/OpenCCU#1830] [🇺🇸 Q&A] [answered]
Hi, i really would like to rename my devices for better overview. I have around 60 devices in my house and reallllyyyy would like to rename the devices. 

I read the wiki but didnt find anything about renaming.

**Comment by MichaelN0815** [comment, marked as answer]
![grafik](https://user-images.githubusercontent.com/68965895/167251165-28547efc-1a26-4901-aa75-fca84904add2.png)

**Reply by jens-maus** [reply]
And see here for even more documentation on using the CCU/RaspberryMatic WebUI:

https://www.eq-3.de/Downloads/eq3/download%20bereich/handbuecher/WebUI_Handbuch_eQ-3.pdf

https://homematic-forum.de/forum/viewtopic.php?f=49&t=45485
